packages/mint-components/src/components/sqm-portal-register/sqm-portal-register.feature

```diff
@@ -99,4 +99,21 @@ Feature: Portal Register
         Examples:
             | mayHave      | value          | redirectPath   |
             | doesn't have | N/A            | /verifyEmail   |
-            | has          | /verifyMyEmail | /verifyMyEmail |
\ No newline at end of file
+            | has          | /verifyMyEmail | /verifyMyEmail |
+
+    @ui
+    Scenario Outline: Terms and conditions link can be included above the register button
```

Comment:
```suggestion
    Scenario Outline: Slotted content can be included above the register button
```
Scenario description should be more generic, while we are building it for T&C text/link, the slot we are adding is  much more general than that